packages/universe-components/src/donationForm/DonationForm.tsx

```diff
@@ -48,25 +66,35 @@ const getFallbackAddress = (chainName?: string): string => {
 
   return chainName ? fallbackAddresses[chainName] || '' : '';
 };
+
 function sleep(ms: number) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
-export default function DonationForm({ initiative }: DonationFormProps) {
+export default function DonationForm({ initiative, rate }: DonationFormProps) {
```

Comment:
Looks like `rate` prop isn't used, should be removed

Reply:
Tried to use it as we were losing it's value on load. For sure we will use it once we fix the issue with initial data. Leave for now.